hadoop-examples/README.md

```diff
@@ -0,0 +1,94 @@
+<!--
+Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+
+# hadoop-connectors examples
+Examples of Hadoop Connectors for Pravega.
+
+Description
+-----------
+These examples give you some basic ideas how to use hadoop-connectors for pravega
+
+Build
+-------
+(Most of these steps will be removed when GAed)
+### build hadoop connectors
+```
+git clone --recurse-submodules https://github.com/pravega/hadoop-connectors.git
+cd hadoop-connectors
+gradle install
+
+# start pravega whose version matches hadoop-connectors
+cd pravega
+./gradlew startStandalone
+```
+
+## build hadoop connectors examples
+```
+open another terminal and goto ~/pravega-samples
+cd hadoop-examples
+gradle build
+```
+
+Run Examples on Local Machine
+---
+
+
+Hadoop (verified with Hadoop 2.8.3 on Ubuntu 16.04)
+```
+1. setup and start hdfs
+
+2. set env variables
+   export HDFS=hdfs://192.168.0.188:9000
+   export HADOOP_EXAMPLES_JAR=build/libs/pravega-hadoop-examples-0.3.0-SNAPSHOT-all.jar
+   export HADOOP_EXAMPLES_INPUT_DUMMY=${HDFS}/tmp/hadoop_examples_input_dummy
+   export HADOOP_EXAMPLES_OUTPUT=${HDFS}/tmp/hadoop_examples_output
+   export PRAVEGA_URI=tcp://192.168.0.188:9090
+   export PRAVEGA_SCOPE=myScope
+   export PRAVEGA_STREAM=myStream
+   export CMD=wordcount # so far, can also try wordmean and wordmedian
+
+3. make sure below dirs are empty
+   hadoop fs -rmr ${HADOOP_EXAMPLES_INPUT_DUMMY}
+   hadoop fs -rmr ${HADOOP_EXAMPLES_OUTPUT}
+
+4. generate words into pravega
+   hadoop jar ${HADOOP_EXAMPLES_JAR} randomtextwriter -D mapreduce.randomtextwriter.totalbytes=32000 ${HADOOP_EXAMPLES_INPUT_DUMMY} ${PRAVEGA_URI} ${PRAVEGA_SCOPE} ${PRAVEGA_STREAM}
+
+5. run hadoop command
+   hadoop jar ${HADOOP_EXAMPLES_JAR} ${CMD} ${HADOOP_EXAMPLES_INPUT_DUMMY} ${PRAVEGA_URI} ${PRAVEGA_SCOPE} ${PRAVEGA_STREAM} ${HADOOP_EXAMPLES_OUTPUT}
+```
+
+
+Additionally, you can run WordCount program (more will be coming soon) on top of [HiBench](https://github.com/intel-hadoop/HiBench)
+```
+0. set same env variables as previous section, and
+   export HADOOP_HOME=~/services/hadoop-2.8.3
+   export HDFS=hdfs://192.168.0.188:9000
+   export INPUT_HDFS="${HADOOP_EXAMPLES_INPUT_DUMMY} ${PRAVEGA_URI} ${PRAVEGA_SCOPE} ${PRAVEGA_STREAM}"
+
+1. fetch/build/patch HiBench
+   gradle wcHiBench
+
+2. prepare testing data
+   ./HiBench/bin/workloads/micro/wordcount/prepare/prepare.sh
```

Comment:
Unfortunately, I could not make it run with HiBench 7.0 at the moment (looks like a configuration problem). Perhaps it is a problem of my environment, but the aspect to consider is that I just followed the instructions (other users might experience similar issues). If I have any advance in this regard, I will provide feedback to you.